src/http/server.rs

```diff
@@ -79,6 +79,7 @@ pub struct Configuration {
     pub http_port: u16,
     pub ctrl_port: u16,
     pub https_port: u16,
+    pub core_id: i32,
```

Comment:
`i32` is not ideal. Use [this](https://github.com/esp-rs/esp-idf-hal/blob/master/src/cpu.rs#L13) instead.